### PR TITLE
Upgrade geth to 1.9.5

### PR DESCRIPTION
**Pull request by adambabik**
This is required to upgrade Whisper to use the fixed PoW algorithm.

Additional changes:
* `contracts/ens` is ported from go-ethereum 1.8 because it was removed in 1.9
* `ULC` got removed due to no maintenance and being not used

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,37 +2,46 @@ module github.com/status-im/status-go
...
 	golang.org/x/text v0.3.2
 	gopkg.in/go-playground/assert.v1 v1.2.1 // indirect
-	gopkg.in/go-playground/validator.v9 v9.29.0
+	gopkg.in/go-playground/validator.v9 v9.29.1
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 )
-
-replace github.com/ethereum/go-ethereum v1.8.27 => github.com/status-im/go-ethereum v1.8.27-status.5
-
-replace github.com/NaySoftware/go-fcm => github.com/status-im/go-fcm v1.0.0-status
```
Comment (pedropombeiro) [inline go.mod:38]: We no longer use `github.com/status-im/go-fcm v1.0.0-status`?

Comment (pedropombeiro) [reply go.mod:38]: Oh, hadn't realized it was just moved to the top.

**Comment by pedropombeiro** [comment]
@adambabik I noticed that with this branch, I'm not able to run tests individually from VS Code, while it was possible on `develop`:

```
Running tool: /usr/local/go/bin/go test -timeout 90s github.com/status-im/status-go/api -run ^(TestStartStopMultipleTimes)$ -v

=== RUN   TestStartStopMultipleTimes
--- FAIL: TestStartStopMultipleTimes (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x1100588]

goroutine 43 [running]:
testing.tRunner.func1(0xc000102b00)
	/usr/local/go/src/testing/testing.go:830 +0x392
panic(0x138a2c0, 0x256e420)
	/usr/local/go/src/runtime/panic.go:522 +0x1b5
github.com/status-im/status-go/t/utils.MakeTestNodeConfig(0x309, 0x46ce48, 0x2d820ade78b, 0x10404101)
	/home/pedro/go/src/github.com/status-im/status-go/t/utils/utils.go:254 +0x198
github.com/status-im/status-go/api.TestStartStopMultipleTimes(0xc000102b00)
	/home/pedro/go/src/github.com/status-im/status-go/api/backend_test.go:389 +0x3c
```

`TestConfig` seems to be nil in `MakeTestNodeConfig`.

**Comment by adambabik** [comment]
@PombeirP please pull the latest changes. The tests in the CI passed for the last commit.

**Comment by pedropombeiro** [comment]
> @PombeirP please pull the latest changes. The tests in the CI passed for the last commit.

@adambabik I know, the tests pass when you run for a whole package, but not when run individually.

Can you try the following?

``` bash
go test -timeout 90s github.com/status-im/status-go/api -run '^(TestStartStopMultipleTimes)$' -v
```

**Comment by adambabik** [comment]
@PombeirP good catch! 

Unfortunately, in Go 1.13 parsing flags in tests changed a bit. We were relying on the order of `init()` function calls which is kinda undefined but worked in our case.

I removed that but if any test function/method depends on `t/utils` it needs to call `unit.Init()`.

**Comment by pedropombeiro** [comment]
@adambabik I saw your 1.9.5 branch in status-react isn't building, and I'm not sure what commit you're pointing to for nixpkgs. I needed to start a branch from 1.9.5 and that one contains the fix for Go 1.12, in case you want to cherry-pick.

File: mobile/status.go [hunk 2]
```diff
@@ -746,15 +745,3 @@ func SignHash(hexEncodedHash string) string {
 
 	return hexEncodedSignature
 }
-
-func GenerateAlias(pk string) string {
```
Comment (pedropombeiro) [inline mobile/status.go]: @adambabik Looks like you deleted these methods by mistake?

Comment (jakubgs) [reply mobile/status.go]: Yeah, you're missing 9a3ed980c9dcbc6e55fcba843224e447b06eb08f, and probably also 40e66e6186164a422e432fd754b8f86b958014f3.

Comment (adambabik) [reply mobile/status.go]: Oh whoops. I rebased against develop and resolved all conflicts using `-s ours`... I should use that only on `vendor/` conflicts. Fixing.

**Pull request merged**